### PR TITLE
[skipci]Doc/guide commit msg

### PR DESCRIPTION
**Pull request by Cyber-SiKu**
Signed-off-by: Cyber-SiKu <Cyber-SiKu@outlook.com>

### What problem does this PR solve?

add commit message required

Issue Number: #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: Community_Guidelines_cn.md [hunk 1]
```diff
@@ -62,6 +62,44 @@ Curve 社区有多个[沟通渠道](#社区交流)，每两周还会有线上的
...
+- 避免长的复合词和缩写
+
+在提交时请尽可能的遵循以下格式：
+
+```
+[type]<scope>: <description>
+<BLANK LINE>
+[body]
+<BLANK LINE>
+[footer]
```
Comment (cw123) [inline Community_Guidelines_cn.md:86]: [footer] 是什么？

Comment (Cyber-SiKu) [reply Community_Guidelines_cn.md:86]: > [footer] 是什么？

update

File: Community_Guidelines_cn.md [hunk 2]
```diff
@@ -62,6 +62,46 @@ Curve 社区有多个[沟通渠道](#社区交流)，每两周还会有线上的
...
+- ci: 影响持续继承相关的功能
+- docs: 文档相关的修改
+- feat: 增加新的特性
+- fix: bug 修复
+- perf: 性能提升
+- refactor: 重构相关的代码，不增加功能也不修复错误
+- style: 不影响代码的的含义的修改，仅仅是修改代码风格
+- test: 单元测试相关的修改
+
+footer 是可选的，用来记录对应因为这些更改而可以关闭的 issue，如 `Close issue-12345`
```
[the remark this answers is not in the document]
Comment (Cyber-SiKu) [reply Community_Guidelines_cn.md:101]: > 这个close issue 的语法好像不对

https://stackoverflow.com/questions/60027222/github-how-can-i-close-the-two-issues-with-commit-message

[the remark this answers is not in the document]
Comment (Cyber-SiKu) [reply Community_Guidelines_cn.md:101]: > 这个close issue 的语法好像不对

close 应该也可以吧

**Review by cw123 — APPROVED** [review]

**Pull request merged**